README.md

```diff
@@ -39,46 +40,62 @@ Concretely, it's a *typed* content address: a tuple of `(content-type, content-a
 
 Current version: CIDv1
 
-A CIDv1 has four parts:
+The atomic CIDv1 is a **binary** format composed from `unsigned_varint`s that encode values from the , each of which :
 
 ```text
-<cidv1> ::= <mb><multicodec-cidv1><mc><mh>
+<cidv1> ::= <CIDv1-multicodec><content-multicodec><multihash>
```

Comment:
"content-multicodec" gets us into tricky territory - we've opted for using the term "multicodec code" to describe the numbers themselves; and specifically here we're talking about an "ipld codec multicodec code"!

I don't really have a better suggestion; but I think we need to explain these terms once introduced, or else we may not be much better off than the current form.